k8s/production/karpenter/provisioners/gitlab/provisioner.yaml

```diff
@@ -19,6 +19,7 @@ spec:
         - "m5.xlarge"
         - "m5.4xlarge"
         - "c5.4xlarge"  # recommended by gitlab - https://docs.gitlab.com/ee/administration/reference_architectures/3k_users.html#cluster-topology
+        - "c6a.8xlarge"
```

Comment:
Maybe update the comment here to explain that this is following the recommendation? Having a comment on the above instance type but not this one implies they aren't related